pkg/build/fuchsia.go

```diff
@@ -92,3 +95,23 @@ func runSandboxed(timeout time.Duration, dir, command string, arg ...string) ([]
 	}
 	return osutil.Run(timeout, cmd)
 }
+
+// genSSHKeys generates a pair of ssh keys inside the given directory, named key and key.pub.
+// If both files already exist, this function does nothing.
+// The function returns the path to both keys.
+func genSSHKeys(dir string) (privKey, pubKey string, err error) {
+	privKey = filepath.Join(dir, "key")
+	pubKey = filepath.Join(dir, "key.pub")
+
+	if !osutil.IsExist(privKey) || !osutil.IsExist(pubKey) {
+		if _, err := osutil.RunCmd(time.Minute, dir, "rm", "-f", privKey, pubKey); err != nil {
```

Comment:
We generally don't shell out to subprocesses for simple actions like this that can be done without shelling out.
```
	os.Remove(sshkey)
	os.Remove(sshkeyPub)
```
should do.

Is there a particular reason to do IsExist checks? There is similar code in akaros.go:
```
	sshkey := filepath.Join(params.KernelDir, "key")
	sshkeyPub := sshkey + ".pub"
	os.Remove(sshkey)
	os.Remove(sshkeyPub)
	if _, err := osutil.RunCmd(10*time.Minute, "", "ssh-keygen", "-t", "rsa", "-b", "2048",
		"-N", "", "-C", "", "-f", sshkey); err != nil {
		return err
	}
```
Would be reasonable to keep consistent.